### PR TITLE
Added Code scan functionality in doc.

### PR DESCRIPTION
**Pull request by SeqLaz**
Added Code scan functionality in documentation

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3064 Document QR and barcode scanner](https://app.clickup.com/t/2192114/QF-3064)

**Comment by m-kuhn** [comment]
Thank you for documenting this awesome functionality by @nirvn

**Review by suricactus — APPROVED** [review]
Thanks a lot! Are you planning to add the search bar functionality in a separate PR? In any case separate PR might make more sense indeed.

**Comment by suricactus** [comment]
Side comment:

Suggesting FOSS (online) tool to generate QR codes might be a good idea?

File: documentation/how-to/search.en.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ open expand the search bar.
 
 The value entered in the search bar will be used to find features with
 matching attribute values. A minimum of three characters is required to
-start the search.
+start the search. Additionally, you have the option to employ the code scan functionality for this purpose.
```
Comment (suricactus) [inline documentation/how-to/search.en.md:21]: I believe here we can elaborate HOW it works and we can suggest encoding your feature UUIDs, so we can find them right away.

Comment (SeqLaz) [reply documentation/how-to/search.en.md]: Done!

**Review by suricactus — APPROVED** [review]
Thanks!

On QR generator, I failed to find a feature rich example that is hosted on GitHub, which I find problematic, so we can skip for now...

**Comment by nirvn** [comment]
(pushing some refreshed images into this PR, please wait an hour prior to merging)

**Comment by nirvn** [comment]
@SeqLaz , screenshots updated, let me know if you are happy, we'll merge then :)

**Comment by SeqLaz** [comment]
> @SeqLaz , screenshots updated, let me know if you are happy, we'll merge then :)

@nirvn, more than happy, great images

**Pull request merged**